src/ripple/app/main/Main.cpp

```diff
@@ -49,6 +49,7 @@
 #include <cstdlib>
 #include <iostream>
 #include <utility>
+#include <stdexcept>
```

Comment:
Yeah, but isn't it a good practice to include headers, even if they already happen to be included?